### DISCUSSION TITLE
What's best practice to invoice single items in addition to (or separate from) tracked time?

### DISCUSSION THREAD

**Discussion by msbt** [kimai/kimai#4670] [General] [answered]
I tried searching for answers on that topic, but couldn't find any, so please forgive me if that was already answered in other places. 

I need to bill my customers yearly/monthly/one-time fees (could be both, recurring or one-time payments for domains, maintenance contracts, third party purchases). So those items don't have working hours attached to it. Is that even possible and if so, which would be the best way to charge for those? Currently I just create an new tracked time and add a fixed cost, but that doesn't seem right.

Best,
M

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/store/expenses-bundle.html
